source/_integrations/weatherflow.markdown

```diff
@@ -0,0 +1,65 @@
+---
+title: "WeatherFlow"
+description: "WeatherFlow weather sensor"
+ha_release: "0.38"
```

Comment:
This seems unlikely 😉

Reply:
I've optimistically changed it to the next release 🤞️